ts/models/conversation.ts

```diff
@@ -2441,16 +2442,11 @@ export class ConversationModel extends Backbone.Model<ConversationAttributes> {
     });
 
     // no need to trigger a UI update now, we trigger a messagesAdded just below
-    const messageId = await model.commit(false);
+    const messageId = await model.commit(true);
```

Comment:
Now we are triggering a UI update? If that is expected then you need to update the comment above.

Reply:
fixed